exercises/math_with_protocols.livemd

```diff
@@ -96,62 +96,88 @@ Define implementations for the `Math` protocol as documented below.
 
 ```elixir
 defprotocol Math do
-  @doc """
-  Add two integers, lists, or strings.
+  def add(value1, value2)
+end
+
+@doc """
```

Comment:
You have an @doc outside of a module.

While this is the same color as a comment, it's not a comment. It's the same as defining any module attribute but the Elixir highlighter is smart enough to make it grey when it's an `@doc` module attribute.